src/funcs.c

```diff
@@ -528,81 +518,50 @@ static ALWAYS_INLINE Obj DoExecFunc(Obj func, Int narg, const Obj *arg, Int isAt
-    Obj                 func )
+static Obj DoExecFunc0args(Obj func)
 {
-    return DoExecFunc(func, 0, 0, 0);
+    return DoExecFunc(func, 0, 0);
 }
 
-Obj             DoExecFunc1args (
-    Obj                 func,
-    Obj                 arg1 )
+static Obj DoExecFunc1args(Obj func, Obj a1)
 {
-    Obj arg[] = { arg1 };
-    return DoExecFunc(func, 1, arg, 0);
+    Obj arg[] = { a1 };
+    return DoExecFunc(func, 1, arg);
 }
 
-Obj             DoExecFunc2args (
-    Obj                 func,
-    Obj                 arg1,
-    Obj                 arg2 )
+static Obj DoExecFunc2args(Obj func, Obj a1, Obj a2)
 {
-    Obj arg[] = { arg1, arg2 };
-    return DoExecFunc(func, 2, arg, 0);
+    Obj arg[] = { a1, a2 };
+    return DoExecFunc(func, 2, arg);
 }
 
-Obj             DoExecFunc3args (
-    Obj                 func,
-    Obj                 arg1,
-    Obj                 arg2,
-    Obj                 arg3 )
+static Obj DoExecFunc3args(Obj func, Obj a1, Obj a2, Obj a3)
 {
-    Obj arg[] = { arg1, arg2, arg3 };
-    return DoExecFunc(func, 3, arg, 0);
+    Obj arg[] = { a1, a2, a3 };
+    return DoExecFunc(func, 3, arg);
 }
 
-Obj             DoExecFunc4args (
-    Obj                 func,
-    Obj                 arg1,
-    Obj                 arg2,
-    Obj                 arg3,
-    Obj                 arg4 )
+static Obj DoExecFunc4args(Obj func, Obj a1, Obj a2, Obj a3, Obj a4)
 {
-    Obj arg[] = { arg1, arg2, arg3, arg4 };
-    return DoExecFunc(func, 4, arg, 0);
+    Obj arg[] = { a1, a2, a3, a4 };
+    return DoExecFunc(func, 4, arg);
 }
 
-Obj             DoExecFunc5args (
-    Obj                 func,
-    Obj                 arg1,
-    Obj                 arg2,
-    Obj                 arg3,
-    Obj                 arg4,
-    Obj                 arg5 )
+static Obj DoExecFunc5args(Obj func, Obj a1, Obj a2, Obj a3, Obj a4, Obj a5)
 {
-    Obj arg[] = { arg1, arg2, arg3, arg4, arg5 };
-    return DoExecFunc(func, 5, arg, 0);
+    Obj arg[] = { a1, a2, a3, a4, a5 };
+    return DoExecFunc(func, 5, arg);
 }
 
-Obj             DoExecFunc6args (
-    Obj                 func,
-    Obj                 arg1,
-    Obj                 arg2,
-    Obj                 arg3,
-    Obj                 arg4,
-    Obj                 arg5,
-    Obj                 arg6 )
+static Obj DoExecFunc6args(Obj func, Obj a1, Obj a2, Obj a3, Obj a4, Obj a5, Obj a6)
 {
-    Obj arg[] = { arg1, arg2, arg3, arg4, arg5, arg6 };
-    return DoExecFunc(func, 6, arg, 0);
+    Obj arg[] = { a1, a2, a3, a4, a5, a6 };
+    return DoExecFunc(func, 6, arg);
 }
 
-Obj             DoExecFuncXargs (
-    Obj                 func,
-    Obj                 args )
+static Obj DoExecFuncXargs(Obj func, Obj args)
 {
     Bag                 oldLvars;       /* old values bag                  */
-    REMEMBER_LOCKSTACK();
     UInt                len;            /* number of arguments             */
     UInt                i;              /* loop variable                   */
```

Comment:
In `DoExecFuncXargs`, the diff is a bit hard to read, because `git` resp. GitHub "merges" it with the (now removed) `DoExecFuncXargsL`. The realy diffs here are: `REMEMBER_LOCKSTACK` was moved move down; and `#ifdef HPCGAP ... LockFuncArgs` has been inserted.